services/dashboard/views/apps/streamingapp/dag.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+
+angular.module('dashboard')
+
+  .config(['$stateProvider',
+    function($stateProvider) {
+      'use strict';
+
+      $stateProvider
+        .state('streamingapp.dag', {
+          url: '/dag',
+          templateUrl: 'views/apps/streamingapp/dag.html',
+          controller: 'StreamingAppDagCtrl'
+        });
+    }])
+
+  .controller('StreamingAppDagCtrl', ['$scope', '$state', '$modal', '$contextmenu', '$visNetworkStyle',
```

Comment:
No, just the controllers that manipulate REST data and their related scope types. There may be situations where changing a controller doesn't provide much benefit, but it would be good to tag those that are candidates so we have agreement and subsequent tasks can focus on one or more controllers.